### DISCUSSION TITLE
Indefinitely retain starred events

### DISCUSSION THREAD

**Discussion by yasahiro-x** [blakeblackshear/frigate#4367] [Ideas]
I've realised that there's an option in the current UI to star particular events, but I can't find any mention of the functionality in the documentation.
It might be a good idea to enable an option such that starred event clips are retained indefinitely and not affected by the usual recording limitations.

**Comment by NickM-27** [comment]
> It might be a good idea to enable an option such that starred event clips are retained indefinitely and not affected by the usual recording limitations.

This is how it works, they are retained indefinitely. Agree that it would be good to add to the docs.
